docs/reference/cc-rest-api/index.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Cleura Cloud provides a REST API, along with an automatically generated documentation page.
+---
+# Cleura Cloud REST API automatically generated documentation page
+
+You may at any time make good use of the {{rest_api}} with a
+tool like `curl`. Regarding documentation, there is a detailed,
+[automatically generated page](https://apidoc.cleura.cloud),
+which you turn to for advice on endpoints and how to use them.
+
+> To access the API, you need to have an
+[account in {{brand}}](../../howto/getting-started/create-account),
+and also
+[create an access token](../../howto/getting-started/accessing-cc-rest-api).
```

Comment:
I can't see anything wrong with it.

Reply:
OK! I asked because I suggested "you need to have an X, and also a Y" and you made it "you need to have an X, and also *create* a Y", which is of course *also* correct — I was just wondering if maybe you had strong feelings about my suggestion being incorrect or sounding bad. If you didn't, and you simply prefer the version with "create", all is well. :)

Reply:
Ah, yes, now I understand :) There are no strong feelings whatsoever. It's that, subconsciously, I think that an account is something you should _have_, but the token is something you need to _create_ for each session. In other words, you get an account _once_, but you _create_ a token over and over again.

Reply:
That makes perfect sense. :)